pkg/cloud/azure/azure_storage.go

```diff
@@ -137,7 +137,7 @@ func (s *azureStorage) Writer(ctx context.Context, basename string) (io.WriteClo
 		defer sp.Finish()
 		_, err := azblob.UploadStreamToBlockBlob(
 			ctx, r, blob, azblob.UploadStreamToBlockBlobOptions{
-				BufferSize: 4 << 20,
```

Comment:
we should re-test azure (by hand I guess) with this before we backport. I have _vague_ recollections of it having some hard coded/magic numbers on these chunks but I don't remember what they were, so I just want to confirm we're not breaking (untested) code in a stable release here.

Reply:
yikes this fell off my radar, running them now and merging.

Reply:
Ran all the tests in `pkg/cloud/azure` and they passed, merging!